spec/models/order_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  let(:order) { FactoryBot.create(:order) }
```

Comment:
```suggestion
  let(:order) { create(:order) }
```
і так в інших місцях теж